src/open_inwoner/conf/test.py

```diff
@@ -2,7 +2,12 @@
 Staging environment settings module.
 This *should* be nearly identical to production.
 """
+import logging
 import os
+import sys
+
+if len(sys.argv) > 1 and sys.argv[1] == "test":
+    logging.disable(logging.CRITICAL)
 
```

Comment:
I'm not sure if this is what we want. Supposedly we still want to see something when a real problem occurs?

Reply:
I changed it so we keep logs for level >= `ERROR`